src/openrct2/config/Config.cpp

```diff
@@ -101,8 +101,7 @@ namespace OpenRCT2::Config
     });
 
     static const auto Enum_DrawingEngine = ConfigEnum<DrawingEngine>({
-        ConfigEnumEntry<DrawingEngine>("SOFTWARE", DrawingEngine::Software),
-        ConfigEnumEntry<DrawingEngine>("SOFTWARE_HWD", DrawingEngine::SoftwareWithHardwareDisplay),
+        ConfigEnumEntry<DrawingEngine>("SOFTWARE", DrawingEngine::SoftwareWithHardwareDisplay),
```

Comment:
Changing the meaning of "SOFTWARE" is not a good idea IMO, please change this back to SOFTWARE_HWD.

Reply:
Agreed, this wouldn't be ideal when switching between versions.

Reply:
#24325